scripts/checkBamBai.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+url=$1
+index_url=$2
+samtools_region=$3
+
+url_valid=$(curl -s $url | head -n 1 | tr '\0' '\n')
+index_valid=$(curl -s $index_url | head -n 1 | tr '\0' '\n')
+if [ -z "$url_valid" ] || [ -z "$index_valid" ]; 
+then
+	exit 1;
+fi
+
+samtools_od view -b $url $samtools_region $index_url | head -n 1
```

Comment:
I'm not sure piping to head makes sense here. Isn't the output binary BAM data?

Reply:
No the -b is for output bam

Reply:
Ok, fixed to output in non-binary w/ below commit 70f3971